update to resources table (#118)
astronomer/cloud-docs: docs/resource-reference-aws.md

| Resource                                                                                             | Description                                                                                                                                                                                                                     | Quantity / Default Size |
| ---------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------- |
| [EKS Cluster](https://aws.amazon.com/eks)                                                            | An EKS cluster is required to run the Astronomer Cloud Data Plane, which hosts the resources and data required to execute Airflow tasks.                                                                                        | 1x                      |

> | [EC2 Instances](https://aws.amazon.com/ec2/instance-types/)                                          | EC2 instances (nodes) power the system and Airflow components (Webserver, Scheduler, Workers). Auto-scaling of EC2 instances will accommodate the need for more Airflow Deployments.                                            | 2x m5.xlarge          |

Reviewer: ```suggestion
| [EC2 Instances](https://aws.amazon.com/ec2/instance-types/)                                          | EC2 instances (nodes) power the system and Airflow components (Webserver, Scheduler, Workers). EC2 instances auto-scale for additional Airflow Deployments.                                            | 2x m5.xlarge          |
```

> | [RDS for PostgreSQL Instance](https://aws.amazon.com/rds/)                                           | The RDS instance will be the primary database of the Astronomer Cloud Data Plane. It will host a metadata database for each Airflow Deployment hosted on the EKS cluster.                                                       | 1x db.r5.large        |

Reviewer: ```suggestion
| [RDS for PostgreSQL Instance](https://aws.amazon.com/rds/)                                           | The RDS instance is the primary database of the Astronomer Cloud Data Plane. It hosts a metadata database for each Airflow Deployment hosted on the EKS cluster.                                                       | 1x db.r5.large        |
```

| [Elastic IPs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/elastic-ip-addresses-eip.html)     | Elastic IPs are required for connectivity with the Control Plane, and other public services.                                                                                                                                    | 2x                      |

> | [Subnets](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html)                         | Subnets are provisioned in 2 different AZ regions for redundancy, with 1 public and 1 private per AZ. Public subnets are required for the NAT and Internet gateways, while private subnets are required for EC2 nodes.          | 2x /26 and 2x /22       |

Reviewer: ```suggestion
| [Subnets](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html)                         | Subnets are provisioned in 2 different [Availability Zones (AZs)](https://aws.amazon.com/about-aws/global-infrastructure/regions_az/) for redundancy, with 1 public and 1 private subnet per AZ. Public subnets are required for the NAT and Internet gateways, while private subnets are required for EC2 nodes.          | 2x /26 and 2x /22       |
```

> | [Internet Gateway](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Internet_Gateway.html)       | Required for connectivity with the Control Plane, and other public services.                                                                                                                                                    | 1x                      |

Reviewer: ```suggestion
| [Internet Gateway](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Internet_Gateway.html)       | Required for connectivity with the Control Plane and other public services.                                                                                                                                                    | 1x                      |
```

| [NAT Gateways](https://docs.aws.amazon.com/vpc/latest/userguide/vpc-nat-gateway.html)                | NAT Gateways translate outbound traffic from private subnets to public subnets.                                                                                                                                                 | 2x                      |
| [Routes](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Route_Tables.html#route-table-routes)  | Routes are necessary to direct network traffic from the subnets and gateways.                                                                                                                                                   | 2x                      |
| [Route Tables](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Route_Tables.html)               | Home for the routes.                                                                                                                                                                                                            | 2x                      |

> | [VPC](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html)                      | Virtual network for launching and hosting AWS resources.                                                                                                                                                                        | 1x /19                  |

Reviewer: Purpose of the `19` here?
